test/test_transforms.py

```
@@ -3845,6 +3846,102 @@ def test_transform_inverse(self):
+        misc = torch.randn(*batch, 1, device=device)
+
+        td = TensorDict(
+            {"misc": misc, "reward": reward, "next": {"done": done}},
+            batch,
+            device=device,
+        )
+        rb.extend(td)
+        sample = rb.sample(20)
+        assert sample["reward_to_go"].shape == (20, 1)
+        assert sample["reward_to_go"].all() != 0
+
+    @pytest.mark.parametrize("gamma", [0.99, 1.0])
+    @pytest.mark.parametrize("done_flags", [1, 5])
+    def test_transform_inverse(self, gamma, done_flags):
+        device = "cpu"
+        batch = [20]
+        torch.manual_seed(0)
+        done = torch.zeros(*batch, 1, dtype=torch.bool)
+        done_flags = torch.randint(0, *batch, size=(done_flags,))
+        done[done_flags] = True
+        reward = torch.randn(*batch, 1, device=device)
+        misc = torch.randn(*batch, 1, device=device)
+        r2g = Reward2GoTransform(gamma=gamma)
+        td = TensorDict(
+            {"misc": misc, "reward": reward, "next": {"done": done}},
+            batch,
+            device=device,
+        )
+        td = r2g.inv(td)
+        assert td["reward_to_go"].shape == (20, 1)
+        assert td["reward_to_go"].all() != 0
+
+    def test_transform(self):
+        raise pytest.skip("No inverse for Reward2Go")
+
+    def test_transform_env(self):
+        raise pytest.skip("No environment transform for Reward2Go")
+
+    def test_parallel_trans_env_check(self):
+        raise pytest.skip("No environment transform for Reward2Go")
+
+    def test_single_trans_env_check(self):
+        raise pytest.skip("No environment transform for Reward2Go")
+
+    def test_serial_trans_env_check(self):
+        raise pytest.skip("No environment transform for Reward2Go")
+
+    def test_trans_serial_env_check(self):
+        raise pytest.skip("No environment transform for Reward2Go")
+
+    def test_trans_parallel_env_check(self):
+        raise pytest.skip("No environment transform for Reward2Go")
```

If any of these guys should raise an exception (see below), it should be tested